aikau/src/main/resources/alfresco/preview/PdfJs/TextLayerBuilder.js

```diff
@@ -300,24 +302,13 @@ define(["dojo/_base/declare",
             offset: undefined
           };
 
-          function beginText(begin, className) {
```

Comment:
These two functions were moved down because they both call a function that was previously defined after them.